src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -22,11 +22,15 @@ TODO
 
 ### Setup 1
 
-TODO
+✏️ Update **src/App.tsx** to the following:
```

Comment:
Use the same words for all of these lines: https://github.com/bitovi/academy/blob/b465c568da18defa464ea136798f479eb0830eb5/src/react-vite/03-intro-to-jsx/intro-to-jsx.md#L226